### PR TITLE
Feat/4ls partners

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-3986
| License          | BSD 3-Clause

## Description

Add display for partners support levels

## Related
- [x] TRANS-26311

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/user/support-level/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
+  "user_account_support_level_section_title_description_standard_premium": "Vous êtes un Standard Partner et vous avez souscrit au support Premium. Ce niveau vous garantit une meilleure qualité de service avec, notamment, un accès privilégié à nos équipes d’assistance ainsi qu’un traitement prioritaire de vos demandes.",
+  "user_account_support_level_section_title_description_standard_premium-accredited": "Vous êtes récemment devenu un Standard Partner. Vous ne pouvez donc plus souscrire au support Premium for Advanced Partner. Cependant, en tant que revendeur à valeur ajoutée et/ou intégrateur système, vous devez souscrire au support Premium. Ce niveau vous garantit un accès privilégié à nos équipes d'assistance, ainsi qu'un traitement prioritaire de vos demandes. À noter que cette obligation contractuelle ne concerne pas votre activité d’infogérance, de conseil ou de formation.",
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/account/user/support-level/translations/Messages_fr_FR.json:8]: Some discussion about used wording is currently going on, so translations might get an update

File: packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.constants.js [hunk 2]
```diff
@@ -170,8 +151,6 @@ export const URLS = {
 
 export default {
   API_MODEL_SUPPORT_LEVEL,
-  AVAILABLE_SUPPORT_LEVEL,
-  FULL_SUPPORT_LEVEL_SUBSIDIARIES,
-  INACTIVE_SUPPORT_LEVEL,
+  SUBSCRIPTON,
```
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.constants.js:154]: SUBSCRIPTION ?

File: packages/manager/modules/models/support/level/PartnerLevel.class.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export default class PartnerLevel {
+  /**
+   *
```
Comment (FredericEspiau) [inline packages/manager/modules/models/support/level/PartnerLevel.class.js:3]: extra WS

File: packages/manager/modules/models/support/level/PartnerLevel.class.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+   * @param {PartnerLevel} Object payload representing a partner level
+   * @param {PartnerLevel.level} String partnership level
```
Comment (FredericEspiau) [inline packages/manager/modules/models/support/level/PartnerLevel.class.js:5]: Maybe you could explain what a level is 

Also, it looks like it's not a string, it's a value of an enum

File: packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js [hunk 5]
```diff
@@ -40,8 +59,26 @@ export default class UserAccountSupportLevelCtrl {
   }
 
   areAllLevelsAvailable() {
-    return FULL_SUPPORT_LEVEL_SUBSIDIARIES.includes(
-      this.currentUser.ovhSubsidiary,
+    return !some(
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js:62]: move this to SupportLevel class?

File: packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js [hunk 6]
```diff
@@ -40,8 +59,26 @@ export default class UserAccountSupportLevelCtrl {
...
+      this.supportLevels,
+      (level) => !level.isAvailable(this.currentUser.ovhSubsidiary),
     );
   }
+
+  isLevelActive(supportLevel) {
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js:68]: move this to SupportLevel class?

File: packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js (hunk 5 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js:62]: I thought of this but it seems strange to pass `supportLevel` parameter or to handle an array of support levels in a model that is supposed to represent one

File: packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js (hunk 6 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js:68]: This does not only depend on `SupportLevel` but also on a `PartnerLevel` so if it should be moved to a class, I'm not sure `SupportLevel` is the one to go with

**Comment by cbourgois** [comment]
Just a conflict on `packages/manager/modules/models/index.js`

**Comment by ovh-cds** [comment]
CDS Report installation#12585.0 ✘  
* Stage 1
  * Install ✘

**Comment by marie-j** [comment]
Rebased to fix conflicts

**Comment by ovh-cds** [comment]
CDS Report installation#13365.0 ✘  
* Stage 1
  * Install ✘

**Comment by marie-j** [comment]
Rebased to fix conflicts

**Pull request merged**